zng/docs/spec.md

```diff
@@ -100,47 +100,55 @@ The ZNG type system comprises the standard set of primitive types like integers,
 floating point, strings, byte arrays, etc. as well as container types
 like records, arrays, and sets arranged from the primitive types.
 
-For example, a ZNG stream representing the single string "hello world"
-looks like this:
+For example, a TZNG stream representing the single string "hello world"
+may look like this:
 ```
-9:hello, world
+#35:string
+35:hello, world
 ```
-Here, the type ID is the integer "9" representing the string type
-(defined in [Typedefs](#typedefs)) and the data value "hello, world"
-is an instance of the string type.
+Here, the first line binds a tag `35` to the ZNG `string` data type
+and the second line references that tag to specify a value of the `string`
+type. TZNG is being shown here for human readability. In the equivalent
+binary ZNG encoding, a single [Value Message](#32-value-messages) can represent
+the string by referencing the type ID of `9` (per the
+[Primitive Types](#5-primitive-types) table) with no leading
+[Typedef](#311-typedefs) required.
 
-ZNG gets more interesting when different data types are interleaved in the stream.
-For example,
+ZNG gets more interesting when different data types are interleaved in the
+stream.  For example, in the following TZNG:
```

Comment:
```suggestion
stream.  For example, consider this TZNG stream:
```
I think the existing sentence fragment reads a little roughly.

Reply:
Agreed.